packages/xod-cli/README.md

```diff
@@ -9,23 +9,265 @@ other packages. The responsibility of `xod-cli` is to parse command line
+$ xodc --help [COMMAND]
+USAGE
+  $ xodc COMMAND
+...
+```
+<!-- usagestop -->
+
+# Flags, aliases, environment variables
+
+Almost any flag can be replaced with the appropriate environment variable.
+For example, instead of `--username` you can declare variable `XOD_USERNAME`.
+
+| Flag         | Alias | Environment variable |
+|--------------|-------|----------------------|
+| --api        |       | XOD_API              |
+| --debug      |       | XOD_DEBUG            |
+| --help       | -h    |                      |
+| --no-build   |       |                      |
+| --on-behalf  |       | XOD_ONBEHALF         |
+| --output     | -o    | XOD_OUTPUT           |
+| --output-dir | -o    | XOD_OUTPUT           |
+| --password   | -p    | XOD_PASSWORD         |
+| --quiet      | -q    |                      |
+| --username   | -u    | XOD_USERNAME         |
+| --version    | -v    |                      |
+| --workspace  | -w    | XOD_WORKSPACE        |
+
+
+# Commands
+<!-- commands -->
+* [`xodc autocomplete [SHELL]`](#xodc-autocomplete-shell)
+* [`xodc help [COMMAND]`](#xodc-help-command)
+* [`xodc publish [options] [entrypoint]`](#xodc-publish-options-entrypoint)
+* [`xodc resave [options] [entrypoint]`](#xodc-resave-options-entrypoint)
+* [`xodc tabtest [options] [project]`](#xodc-tabtest-options-project)
+* [`xodc transpile [options] [entrypoint]`](#xodc-transpile-options-entrypoint)
+
+## `xodc autocomplete [SHELL]`
+
+display autocomplete installation instructions
+
+```
+USAGE
+  $ xodc autocomplete [SHELL]
+
+ARGUMENTS
+  SHELL  shell type
+
+OPTIONS
+  -r, --refresh-cache  Refresh cache (ignores displaying instructions)
+
+EXAMPLES
+  $ xodc autocomplete
+  $ xodc autocomplete bash
+  $ xodc autocomplete zsh
+  $ xodc autocomplete --refresh-cache
+```
+
+_See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v0.1.0/src/commands/autocomplete/index.ts)_
+
+## `xodc help [COMMAND]`
+
+display help for xodc
+
+```
+USAGE
+  $ xodc help [COMMAND]
+
+ARGUMENTS
+  COMMAND  command to show help for
+
+OPTIONS
+  --all  see all commands in CLI
+```
+
+_See code: [@oclif/plugin-help](https://github.com/oclif/plugin-help/blob/v2.1.3/src/commands/help.ts)_
+
+## `xodc publish [options] [entrypoint]`
+
+publish a library
+
+```
+USAGE
+  $ xodc publish [options] [entrypoint]
+
+ARGUMENTS
+  ENTRYPOINT
+      multipositional argument, reference to project and patch:
+      if omitted:
+      	if current working directory is a project root, then use this project and @/main patch
+      	if current working directory is a patch directory, then use this project and this patch
+      if one position:
+      	if path to a patch directory or file, then use this project and this patch
+      	if path to a project/xodball, then use this project and @/main patch
+      if two positions:
+      	first position is the path to a project/xodball
+      	second position is the patch name like 'main' or '@/main'
```

Comment:
A two-positional entry point makes no sense for `xodc publish`. Should actually be a `[project]`